arbos/arbosState/initialize.go

```diff
@@ -154,16 +154,17 @@ func initializeRetryables(rs *retryables.RetryableState, initData statetransfer.
 
 func initializeArbosAccount(statedb *state.StateDB, arbosState *ArbosState, account statetransfer.AccountInitializationInfo) error {
 	l1pState := arbosState.L1PricingState()
+	posterTable := l1pState.BatchPosterTable()
 	if account.AggregatorInfo != nil {
-		err := l1pState.SetAggregatorFeeCollector(account.Addr, account.AggregatorInfo.FeeCollector)
+		isPoster, err := posterTable.ContainsPoster(account.Addr)
 		if err != nil {
 			return err
 		}
-	}
-	if account.AggregatorToPay != nil {
-		err := l1pState.SetUserSpecifiedAggregator(account.Addr, account.AggregatorToPay)
-		if err != nil {
-			return err
+		if !isPoster {
+			_, err = posterTable.AddPoster(account.Addr, account.AggregatorInfo.FeeCollector)
```

Comment:
As discussed, I don't think we want to copy poster info from Arbitrum Classic

Reply:
Changed this to ignore poster info from classic, except that if the already-existing Nitro batch poster has specified a fee collector address on classic, we copy that in.